addons/respawn/initSettings.sqf

```diff
@@ -0,0 +1,30 @@
+[
+    QGVAR(savePreDeathGear),
+    "CHECKBOX",
+    [LSTRING(SavePreDeathGear_DisplayName), LSTRING(SavePreDeathGear_Description)],
+    LSTRING(DisplayName),
+    true,
```

Comment:
default value should be same as before: false

Reply:
copy paste fail